recipes/gbmunge/meta.yaml

```diff
@@ -0,0 +1,32 @@
+{% set name = "gbmunge" %}
+{% set version = "2018.07.06" %}
```

Comment:
If you created this version, use something like `0.0.2018.07.06`. That way it's sure to be less than whatever versioning scheme upstream might decide in the future. If they release `1.0`, well, `1.0 < 2018.07`, so it won't get installed. (At least not without setting epoch, which I've never tried on conda).